src/styles/index.scss

```diff
@@ -1,3 +1,128 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
```

Comment:
Consider using SCSS variables for properties like `font-family`, `background-color`, and other repeated values. This will help maintain consistency and make it easier to update styles globally.